sp3/src/tests/parser_3d.rs

```diff
@@ -82,15 +82,15 @@ mod test {
         for (epoch, sv, clock) in sp3.sv_clock() {
             assert_eq!(epoch, Epoch::from_str("2019-10-27T00:00:00 GPST").unwrap());
             if sv == sv!("C01") {
-                assert_eq!(clock, 63.035497, "bad clock data");
+                assert!((clock - 63.035497E-6).abs() < 1E-9, "bad clock data");
```

Comment:
Not sure it's worth using it, but https://docs.rs/approx.

Reply:
Yeah I should look into adding this (only in test mode )